### DISCUSSION TITLE
What is better to use with AirSim: Visual Studio Community, Professional or Enterprise?

### DISCUSSION THREAD

**Discussion by catproof** [microsoft/AirSim#4003] [Support Q&A] [answered]
See title! Community is the free version of Visual Studio. Are there any advantages to using the other versions when working with AirSim?...

**Comment by zimmy87** [comment, marked as answer]
I've been using community to run and edit AirSim source and haven't run into a requirement for any features exclusive to professional or enterprise, so I think you should be fine with just using community
